package.json

```diff
@@ -46,6 +50,7 @@
     "test": "npm-run-all test:*",
     "test:karma": "karma start",
     "test:lint": "eslint .",
-    "test:mocha": "istanbul cover _mocha --report lcovonly -- tests/*.test.js -R spec"
+    "test:mocha": "istanbul cover _mocha --report lcovonly -- tests/*.test.js -R spec",
+    "clientize": "webpack"
```

Comment:
Just for my own education, how would this work?
Should this `clientize` script be called via some npm "prepublish" hook or something? Otherwise I guess @jaredkerim will need to manually run it every time he wants to create a new build.